src/store/ChartStore.js

```diff
@@ -147,6 +148,7 @@ class ChartStore {
         chartSetting.onSettingsChange = onSettingsChange;
         this.isMobile = isMobile;
         this.state = this.mainStore.state;
+        this.supportsAnimation = supportsAnimation;
```

Comment:
this should be in `ChartState`.

 I think it is safe to assume that library user would set this once and not go back and forth with this setting - therefore we can stop `animateChart(stxx, { stayPut: true });` and `this.stx.append('draw', this.updateSpot);` from bring called altogether.

Also do add this setting to documentation + explain why we want to disable it.

Reply:
@brucebinary yes this also will work if we wanna disable it all together, though i think its better to follow their documentation https://documentation.chartiq.com/CIQ.Animation.html#main.

Reply:
> Keep in mind that changing to a different chart type, may once again enable animation. You can override this by adding an event listener on layout changes.

...that's too much hassle - considering our use case.

Reply:
@brucebinary sure. i already used your approach!